ms2deepscore/models/load_model.py

```diff
@@ -28,10 +31,58 @@ def load_model(filename: Union[str, Path]) -> SiameseSpectralModel:
         print(f"The model version ({model_settings['version']}) does not match the version of MS2Deepscore "
               f"({__version__}), consider downloading a new model or changing the MS2Deepscore version")
     # Extract model parameters from the checkpoint
-    model_params = model_settings['model_params']
+    model_params = model_settings["model_params"]
 
     # Instantiate the SiameseSpectralModel with the loaded parameters
     model = SiameseSpectralModel(settings=SettingsMS2Deepscore(**model_params))
-    model.load_state_dict(model_settings['model_state_dict'])
+    model.load_state_dict(model_settings["model_state_dict"])
     model.eval()
     return model
+
```

Comment:
The models are now saved and loaded independently. We could in principle also store all three models in one file. Just having a embedding_evaluator_state_dict and a ms2deepscore_state_dict. Saving and loading them in one file would reduce the risk of accidental mismatching models that are not compatible. But it could also be confusing to have multiple models stored in one file...

Reply:
In that case you could still have the 3 separate loading functions btw. And we could handle it in the MS2Deepscore class to make sure the separate three models are all loaded from the same file.

Reply:
No need to implement in this PR, so if you think this would make sense we could make an issue for this.

Reply:
Agree, that would we worth checkout out --> #204